pyclam/datasets.py

```diff
@@ -30,7 +30,7 @@ def ring_data(n: int, radius: float, noise: float) -> np.ndarray:
     return np.asarray(ring, dtype=np.float64)
 
 
-def bullseye(n: int = 1_000, num_rings: int = 3, noise: float = 0.05) -> Tuple[np.ndarray, List[int]]:
+def bullseye(n: int = 2_000, num_rings: int = 3, noise: float = 0.05) -> Tuple[np.ndarray, List[int]]:
```

Comment:
dataset changes shouldn't be part of this PR, not that it really matters, but try and keep them singular in purpose

Reply:
I kinda needed to to be able to measure any speedups